ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/NebulaBasicDaoTests.java

```diff
@@ -501,6 +501,12 @@ public void listStartNodes() {
     System.out.println(JSON.toJSONString(personList));
   }
 
+  @Test
+  public void listEndNodes() {
+    List<Person> personList = repository.listEndNodes("易小海", Like.class);

```

Comment:
Shoud we default think that the user know `endType` of endNodes? @CorvusYe

Reply:
Like listStartNodes, listEndNodes provides two apis where you can specify the type of endNode. 
Do I understand your question correctly?

Reply:
> Shoud we default think that the user know `endType` of endNodes? @CorvusYe 

@shbone 
Like listStartNodes, listEndNodes provides two apis where you can specify the type of endNode. 
Do I understand your question correctly?

Reply:
Yes，one is that you can specify the type of endNode, another is that you dont't know the type of endNode tha you  just know the type of startNode and edge.